test/integration/testdata/ingress.yaml.tmpl

```diff
@@ -3,7 +3,7 @@ kind: Ingress
 metadata:
   name: secured-ingress
   annotations:
-    kubernetes.io/ingress.class: istio
+    kubernetes.io/ingress.class: {{ .IstioNamespace }}
```

Comment:
Nice idea for restricting which ingresses are handled by the controller, but a bit unorthodox, the class is supposed to be something like "istio", "ngnx", "traeffik".

Can we add another annotation for the restriction and keep the original one as "istio"? Still in 0.2, since it will be harder to change this later.

Reply:
I understand the above would require a few more lines in the controller.

Reply:
It's set `istio` by default. This test overrides it to a different value.

Reply:
istio/istio ingress template must be changed in the same commit as that of upgrading pilot to this SHA. It has to be done manually, do not rely on the bot.